esx_service/utils/vmdk_utils.py

```diff
@@ -305,6 +309,28 @@ def get_vm_config_path(vm_name):
     vm_config_path = os.path.join(datastore_path, path)
     return vm_config_path
 
+def check_volumes_mounted(vm_list):
+    """
+    Return error_info any vm in @param vm_list have docker volume mounted
+    """
+    for vm_id in vm_list:
+        vm = vmdk_ops.findVmByUuid(vm_id)
+        for d in vm.config.hardware.device:
+            # if device is not a virtual disk, skip this device
+            if type(d) != vim.vm.device.VirtualDisk:
+                continue
+
+            # Filename format is as follows:
+            # "[<datastore name>] <parent-directory>/tenant/<vmdk-descriptor-name>"
+            # Trim the datastore name and keep disk path.
+            _, disk_path = d.backing.fileName.split(None, 1)
```

Comment:
This cannot trim the datastore correctly if whitespaces are included in the datastore name.

>>> fileName = "[datastor e1] /dockvols/tenant1/vol1.vmdk"
>>> 
>>> _, disk_path = fileName.split(None, 1)
>>> print disk_path
e1] /dockvols/tenant1/vol1.vmdk

Use "]" as a separator.